src/aio/ctx.c

```diff
@@ -64,9 +64,9 @@ void nn_ctx_leave (struct nn_ctx *self)
     }
 
     /*  Notify the owner that we are leaving the context. */
-    if (nn_fast (self->onleave != NULL))
+    if (nn_fast (self->onleave != NULL)) {
```

Comment:
This is a style change.  While I personally like having the extra explicit braces, it isn't intrinsic to this change.  We should remove this for now.  (I'm avoiding style changes to the code base, even though I hate the current style.)